engine/installation/cloud/overview.md

```diff
@@ -4,15 +4,9 @@ aliases:
 description: Installation instructions for Docker on cloud.
 keywords:
 - cloud, docker, machine, documentation,  installation
-menu:
-  main:
-    parent: install_cloud
-    weight: -3
 title: Choose how to install
```

Comment:
"Choose an installation method" (eschewing "your" for consistency, see upthread) sounds good - let's go with that